### PR TITLE
vcproj cleanup, CRAFTALVL, advanced stats update for forms.

### PR DESCRIPTION
**Pull request by TheIrateSeagoer**

### CODE REVIEW

**Comment by Canite** [comment]
Do you have an example lootfilter lines I can use to test this out?

**Comment by TheIrateSeagoer** [comment]
///// Craft Levels
////////////////////////////////////////////////////////////////////////////////

ItemDisplay[MAG]:%SAGE%%CRAFTALVL% %BLUE%%NAME%%CONTINUE% //

ItemDisplay[MAG CRAFTALVL>71]:%NAME% %ORANGE%{100% (4 affixes)}//
ItemDisplay[MAG CRAFTALVL<71 CRAFTALVL>50]:%NAME%{%ORANGE%80% (3 affix)%NL%20% (4 affix)}//
ItemDisplay[MAG CRAFTALVL<51 CRAFTALVL>30]:%NAME%{%ORANGE%60% (2 affix)%NL%20% (3 affix)%NL%20% (4 affix)}//
ItemDisplay[MAG CRAFTALVL<31]:%NAME%{%ORANGE%40% (1 affix)%NL%20% (2 affix)%NL%20% (3 affix)%NL%20% (4 affix)}//

File: BH/Drawing/Stats/StatsDisplay.cpp [hunk 1]
```diff
@@ -1,505 +1,938 @@
...
+    { kWolfTxtFileNoAlias, { 6, 14, 26, 40, 60, 95, 157 } },
+    { kBearTxtFileNoAlias, { 7, 15, 26, 40, 63, 99, 163 } },
+    { kSorceressLightningFCRAlias, { 7, 15, 23, 35, 52, 78, 117, 194 } },
+    { MERC_A1, { 0 } },
+    { MERC_A2, { 0 } },
+    { MERC_A3, { 8, 15, 26, 39, 58, 86, 138 } },
+    { MERC_A5, { 0 } },
+};
+
+std::map<DWORD, std::vector<int>> increased_attack_speed_frames = {
```
Comment (Canite) [inline BH/Drawing/Stats/StatsDisplay.cpp:57]: I don't quite understand what this is, just a static number based on character class? If it's unfinished, we should remove the display in the advanced stat screen. If we keep it, it needs to be renamed from "IAS" to something else

Comment (TheIrateSeagoer) [reply BH/Drawing/Stats/StatsDisplay.cpp:57]: This stub was put in to handle IAS breakpoints for the act 2 mercenary originally, and was going to expand to each character class. I still have yet to find a comprehensive list of min frames/max frames breakpoints for every skill. In the future the static number could be removed in favor of breakpoints in the same way that FCR changes based on the ability.

**Pull request merged**